### PR TITLE
es6 changes for promise and reflect properties

### PR DESCRIPTION
**Pull request by velusgautam**
ES6 update for reflect and promise files in js-examples

### CODE REVIEW

**Comment by velusgautam** [comment]
@wbamberg Can you please verify and merge

File: live-examples/js-examples/reflect/reflect-deleteproperty.html [hunk 1]
```diff
@@ -12,6 +12,6 @@
 Reflect.deleteProperty(array1, '3');
 
 console.log(array1);
-// expected output: Array [1, 2, 3, , 5]
+// expected output: Array(5) [ 1, 2, 3, undefined, 5 ]
```
Comment (wbamberg) [inline live-examples/js-examples/reflect/reflect-deleteproperty.html]: Again, this comment change is not quite accurate - the result shown in the interactive example's console here is: `Array [1, 2, 3, undefined, 5]`.

File: live-examples/js-examples/promise/promise-constructor.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 });
 
 console.log(promise1);
-// expected output: [object Promise]
+// expected output: Promise { <state>: "pending" }
```
Comment (wbamberg) [inline live-examples/js-examples/promise/promise-constructor.html]: The value for `expected output` should be the result shown in the interactive example console, not in the browser's JS console. In this case the example displays the array as: `[object Promise]`, so this comment change is not quite accurate.

(Ideally we would have tests for this, but that's quite hard.)

Comment (velusgautam) [reply live-examples/js-examples/promise/promise-constructor.html]: I have made the changes as requested. Thank you for the feedback

File: live-examples/js-examples/reflect/reflect-deleteproperty.html (hunk 1 above)
Comment (velusgautam) [reply live-examples/js-examples/reflect/reflect-deleteproperty.html]: I have made the changes as requested.

**Comment by velusgautam** [comment]
I have made the changes you have requested. Please have a check @wbamberg

**Pull request merged**